### DISCUSSION TITLE
CUY ANIMELIST SEMESTER 4 SESI 1

### DISCUSSION THREAD

**Discussion by desta-anugra** [deaafrizal/forum#41] [Troubleshooting Room] [answered]
![293453374-b190a302-6722-473e-8501-15f9c51fc29b.png](https://github.com/deaaprizal/forum/assets/50786665/e86548c7-01b1-43b2-b38f-ee558cb65d32)

Kenapa error di bagisn webg.image_url padahal semuanya sudah sesuai sama bg dea butuh bener bg🥲

**Comment by andikaalakate** [comment, marked as answer]
Ganti webg jadi webp, bang
